src/components/footer/index.module.scss

```diff
@@ -11,24 +11,111 @@
     font-size: calc(40% + 0.5vw);
 };
 
-.bg_footer {
-    width: 100vw;
-    height: 100%;
+.bgFooter {
     background-image: url('./bg-footer.svg');
     background-size: 100%;
     padding-bottom: 1%;
+    display: flex;
+    margin: 0 auto;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    .banner {
+        flex-basis: 10%;
+        // padding-left: 10%;
+    }
     ul {
+        flex-basis: 40%;
         list-style-type: none;
-        padding: 4vw 0 0 28vw;
+        padding: 2vw 0 0 0vw;
         li {
-            font-size: calc(30% + 0.5vw);
-            margin-bottom: calc(1% + 0.1vw);
-        };
-        .title_bold {
-            font-size: calc(40% + 0.5vw);;
-            font-weight: bold;
+            // font-size: calc(30% + 0.5vw);
             margin-bottom: calc(1% + 0.1vw);
         };
     };
+    .colQrCode {
+        flex-basis: 150px;
+        padding: 2vw 0 0 0vw;
+        svg {
+            margin-left: 20px;
+        }
+    }
+
+    .lisrContract {
+        flex-basis: 10%;
+        padding: 2vw 0 0 2vw;
+        .squareLine {
+            display:flex;
+            align-items: center;
+            height: 30px;
+            background-color: #21B54D;
+            border-radius: 25px;
+            color: white;
+            padding: 3px 10px;
+            font-size: 14px;
+            position: relative;
```

Comment:
`.squareLine` and `squareFace` is not so fucking different. the only thing different is the background-color?  Use class the way it should be. STOP REPEATING YOURSELF OVER AND OVER. CLEAN IT.